Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx

```diff
@@ -197,13 +211,27 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
             </TooltipHost>
           </div>
         </Flex.Item>
-        <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }} >
+        <Flex.Item size="size.quarter" variables={{ 'size.quarter': '16%' }} >
+          <Text
+            truncated
+            className="semiBold"
+            content={message.sendingStartedDateTime}
+          />
+        </Flex.Item>
+        <Flex.Item size="size.quarter" variables={{ 'size.quarter': '16%' }} >
           <Text
             truncated
             className="semiBold"
             content={message.sentDate}
           />
         </Flex.Item>
+        <Flex.Item size="size.quarter" variables={{ 'size.quarter': '16%' }} >
+          <Text
+            truncated
+            className="semiBold"
+            content={message.sendingDuration}
```

Comment:
We are going to change it and have the front end do the calculation for the "duration" instead of the back end.

Have the calculation resilient to:
sendingStartedDate, sentDate, or both not being there (should just show an empty string in any location if that is the case)

Reply:
moved the calculation to font end